native_client/ctcdecode/ctc_beam_search_decoder.cpp

```diff
@@ -16,31 +16,23 @@ using FSTMATCH = fst::SortedMatcher<fst::StdVectorFst>;
 
 std::vector<std::pair<double, Output>> ctc_beam_search_decoder(
     const std::vector<std::vector<double>> &probs_seq,
-    const std::vector<std::string> &vocabulary,
+    const Alphabet &alphabet,
     size_t beam_size,
     double cutoff_prob,
     size_t cutoff_top_n,
-    size_t blank_id,
     Scorer *ext_scorer) {
   // dimension check
   size_t num_time_steps = probs_seq.size();
   for (size_t i = 0; i < num_time_steps; ++i) {
     VALID_CHECK_EQ(probs_seq[i].size(),
-                   vocabulary.size(),
+                   alphabet.GetSize()+1,
```

Comment:
Why the +1 here ? We have a label for the space, don't we count it?

Reply:
+1 is for the CTC blank label.